recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
     - 0108-Fixed-upb.patch
     - 0109-Add-include-path-for-rpc-headers-from-libtirpc-devel.patch   #[ppc_arch == 'p10']
     - 0301-Updated-protobuf-to-4.21.patch
-    - 0302-Fix-libwebp-linking-errors.patch
+#    - 0302-Fix-libwebp-linking-errors.patch
```

Comment:
Pls remove this and corresponding patch.

Reply:
Its Done